### PR TITLE
Support number properties

### PR DESCRIPTION
**Pull request by justinvdm**
We need to be able to do this:

```
### Block Rar
foo: 23
bar[number]: 23.23
```

### CODE REVIEW

**Comment by justinvdm** [comment]
Depends on #5, otherwise ready for review.

File: src/parser.pegjs [hunk 1]
```diff
@@ -98,11 +99,30 @@ symbol 'symbol'
...
 
 
+point = '.'
+sign = minus / plus
+e = [eE]
+exp = e (minus / plus)? digit+
+frac = point digit+
+minus = '-'
+plus = '+'
+zero = '0'
```
Comment (hodgestar) [inline src/parser.pegjs]: We don't seem to be using `zero`?

**Comment by hodgestar** [comment]
One minor comment, otherwise looks good.

**Comment by hodgestar** [comment]
:+1:

**Comment by justinvdm** [comment]
Ready for re-review.

**Pull request merged**